components/AppNav.tsx

```diff
@@ -25,39 +27,38 @@ const dropdownMenuItems = [
 
 const NavBar: React.FC<AuthLinkProps> = ({ session }) => {
   const isAdmin = _.get(session, 'user.isAdmin', '')
+  //window object is not avaible in server side rendering phase
+  const location: string =
+    window && '/' + window.location.pathname.split('/')[1]
```

Comment:
Without first check next will print errors (since static bundling is done server side and there is no window object). Splitting path by `/` will always produce array with more than one item.

Reply:
Should get the location from `router` instead

Reply:
What are advantages of getting pathname from router? Because if I use router I need to change every test which uses this component otherwise they will fail because router is null.